analyses/org.osate.analysis.flows/src/org/osate/analysis/flows/FlowLatencyAnalysisSwitch.java

```diff
@@ -788,10 +921,10 @@ public AnalysisResult invoke(ComponentInstance ci) {
 	 * @param bestCaseEmptyQueue Assume empty queue (instead of full)
 	 * @return A populated report in AnalysisResult format.
 	 */
-	public AnalysisResult invoke(ComponentInstance ci, SystemOperationMode som, boolean asynchronousSystem,
+	private AnalysisResult invoke(ComponentInstance ci, SystemOperationMode som, boolean asynchronousSystem,
```

Comment:
"invoke" methods are meant to be used by other plugins so they should be public.